configs/projects/_shared-agent-components.rb

```diff
@@ -27,9 +27,9 @@
 
 proj.component 'ruby-augeas' unless platform.is_windows?
 proj.component 'ruby-shadow' unless platform.is_aix? || platform.is_windows?
-# We only build ruby-selinux for EL 5-7
-if platform.is_el? || platform.is_fedora?
-  proj.component 'ruby-selinux'
+# We only build ruby-selinux for EL, Fedora, Debian and Ubuntu (amd64/i386)
+if platform.is_el? || platform.is_fedora? || platform.name =~ /debian|ubuntu/
+  proj.component 'ruby-selinux' if platform.name !~ /ubuntu-.*-ppc64el/
```

Comment:
To build this library on a cross-compiled Ubuntu (ppc64el) we would need `pl-build-tools` packages that also ship libselinux and its headers. Currently this only happens when cross-compiling with EL:

```
[root@uxwlicc4x8osecu ~]# find /opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/ | grep selinux
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/av_permissions.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/avc.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/context.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/flask.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/get_context_list.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/get_default_type.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/label.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/restorecon.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/selinux/selinux.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/include/linux/selinux_netlink.h
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/lib/libselinux.so
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/lib/libselinux.so.1
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/lib64/libselinux.so
/opt/pl-build-tools/ppc64le-redhat-linux/sysroot/usr/lib64/libselinux.so.1
```